src/main/java/no/nav/foreldrepenger/soknad/server/app/swagger/FrontendOpenApiResource.java

```diff
@@ -0,0 +1,39 @@
+package no.nav.foreldrepenger.soknad.server.app.swagger;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import io.swagger.v3.oas.integration.api.OpenApiContext;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import no.nav.foreldrepenger.soknad.server.app.ApiConfig;
+
+@ApplicationScoped
+@Path("/frontend/openapi.{type:json|yaml}")
+public class FrontendOpenApiResource {
+    private static final String SWAGGER_ID = "frontend";
+    private static final OpenApiContext openApiContext = new OpenApiKonfigurasjon(SWAGGER_ID, null)
+        .resourceClasses(ApiConfig.getApplicationClasses())
+        .readerClass(OpenApiReaderTypeGeneringFrontend.class)
+        .buildContext();
+
+
+    @GET
+    @Produces({MediaType.APPLICATION_JSON, "application/yaml"})
```

Comment:
Samme kommentar som forrige. OpenApiResource har dette for deg.
Hvis du har de 2 OpenApi i hver sin JaxRs-app så blir det ikke noe konflikt